### ISSUE TITLE
"Fauxbar Lite: Add as search engine..." quietly fails

### ISSUE THREAD

**Issue by Chealer** [ChrisNZL/Fauxbar#72]
The "Fauxbar Lite: Add as search engine..." context menu entry offered in search fields like the one on [Wikipedia](https://en.wikipedia.org/wiki/ASP.NET_Core) fails to work in Google Chrome 107.0.5304.107. It fails quietly, without creating a new search engine.

This was reported on Fauxbar Lite reviews as early as 2019.

![Fauxbar Lite capture](https://user-images.githubusercontent.com/486141/204318942-fcc64ab6-4c7c-4f7e-9935-bd21d91eb3dc.png)

**Comment by ChrisNZL** [comment]
Yeah, Fauxbar (and Fauxbar Lite) hasn't received any updates since 2019 due to:
- Lack of motivation
- Chronic health issues on my end
- WebSQL being deprecated
- and lately, Chrome Manifest V3 being required coming up, which will render Fauxbar completely dead.

I've dabbled with refactoring Fauxbar to just use JavaScript arrays and objects, without relying on a database, but the whole project is so intertwined and built around WebSQL, that it is probably most feasible for Fauxbar to be rebuilt from scratch.

As such, yep it's broken in places, and will probably remain broken, and the whole extension will stop working soon in 2023.

Don't want to close this GitHub (freeze on ice), but I'd recommend not getting attached to anything.

Time will tell if I get so frustrated at Fauxbar not working _at all_ in the coming months that it motivates me to do something about it again, but I do not know yet.

**Comment by Chealer** [comment]
Thank you Chris

Considering the importance of this bug and its persistence, can someone with the necessary permissions "pin" this ticket?
